dpg-post-codegen.md

```diff
@@ -0,0 +1,124 @@
+# After Generate Client SDK via Data-Plane Generator
+
+## Documentation
+
+### README.md
+
+Generator typically helps generate an outline of start-up guide as `README.md` in the output folder.
+Upon SDK publish, the content of the `README.md` will be synchronized to [Microsoft Docs site](https://docs.microsoft.com/), and to respective language sites as well.
+
+It is important to understand that automated generator does not have the domain knowledge to make the `README.md` easy to use for first-time user.
+Therefore, it is highly recommended for developer to enhance the `README.md`.
+
+How to:
+- [Java](https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods-Quickstart-with-AutoRest#improve-sdk-documentation)
+
+### Samples
+
+Besides the start-up guide as `README.md`, user welcomes code samples of variety use cases, from code snippet of API call, to end-to-end champion scenario.
+Upon SDK publish, a properly configured samples folder will be synchronized to [Samples Browse](https://docs.microsoft.com/samples/browse/).
+
+TODO (weidxu): there would be a shared guideline on `samples/README.md` and its metadata, for publishing to Samples Browse.
+
+How to:
+- [Java](https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods-Quickstart-with-AutoRest#samples)
+
+### API documentation
+
+Generator generates API documentation as docstring in code.
+
+Though all API documentation will be published to Microsoft Docs site, user of language SDK typically use API documentation from language sites.
+Therefore, it is helpful for developer to review the API documentation, see it as user will see after SDK publish.
```

Comment:
Is this part to notice users that they could customize docstring in code (like [docstring](https://github.com/Azure/azure-sdk-for-python/blob/4b8a195e4c06946cb8ecea0d40eda049de04ca2b/sdk/appcontainers/azure-mgmt-appcontainers/azure/mgmt/appcontainers/operations/_certificates_operations.py#L234-L244) for better usage experience of customers

Reply:
It is kind of preview for
- java https://azuresdkdocs.blob.core.windows.net/$web/java/azure-iot-deviceupdate/1.0.0-beta.2/com/azure/iot/deviceupdate/DeviceManagementClient.html
- python https://azuresdkdocs.blob.core.windows.net/$web/python/azure-iot-deviceupdate/1.0.0b2/azure.iot.deviceupdate.operations.html#azure.iot.deviceupdate.operations.DeviceManagementOperations (<-- I am not sure whether it is this)

Reply:
On second thought, we might just remove this topic.

If I am Java dev, this is the first thing I want to see, as this is what user sees on the lib.

But if I am service, I won't care...